Applications/ConsumerApi/test/ConsumerApi.Tests.Integration/Features/Onboarding/GET.feature

```diff
@@ -0,0 +1,22 @@
+@Integration
+Feature: GET /resource/resourceId?appName=appName#secretKey endpoint
+
+User requests onboarding page
+
+    Scenario: User agent is "iPhone" and only one app-store link should be contained on the response page
+        Given a http client with user agent iPhone
+        When a call is made to the /resource/resourceId endpoint for the enmeshed app
```

Comment:
I would suggest to rephrase it like this:
```suggestion
        When a call is made to the /resource/resourceId endpoint for the app "enmeshed"
```

In my opinion, this makes it much clearer what the important part is.